lib/ExpensiMark.js

```diff
@@ -121,18 +122,32 @@ export default class ExpensiMark {
             },
 
             /**
-             * Converts markdown style images to img tags e.g. ![Expensify](https://www.expensify.com/attachment.png)
+             * Converts markdown style images and videos to image/video tags e.g. ![Expensify](https://www.expensify.com/attachment.png)
              * We need to convert before linking rules since they will not try to create a link from an existing img
              * tag.
              * Additional sanitization is done to the alt attribute to prevent parsing it further to html by later
              * rules.
              */
             {
-                name: 'image',
+                name: 'imageOrVideo',
```

Comment:
NAB: We might consider breaking this down into two separate rules: image and video. By doing this, we could differentiate between videos and images using the specified REGEX.

Anyway, the current code also looks good to me

Reply:
I agree that this should be split into two different rules if that's easily doable.